### PR TITLE
Add CSS block extension

### PR DESCRIPTION
**Pull request by milljm**
Allow for CSS to control markdown paragraphs.

Refs: #7716

### CODE REVIEW

File: docs/content/utilities/documentation/moose_flavored_markdown.md [hunk 1]
```diff
@@ -214,6 +216,31 @@ Markdown allowing an input code block to float to the right:
 !text test/tests/kernels/simple_diffusion/simple_diffusion.i start=Kernels end=Executioner float=right padding-left=20px font-size=smaller
 ```
 
+### Block CSS Options
+You can apply a style sheet to a markdown paragraph through the use of !css extension:
+
+```markdown
+!css font-size=smaller margin-left=70% color=red text-shadow=1px 1px 1px rgba(0,0,0,.4)
+This paragraph should be of a smaller red font with a black offset text shadow. The text
+wil be aligned to the right due to the margin-left attribute (nice trick to preserve the
```
Comment (milljm) [inline docs/content/utilities/documentation/moose_flavored_markdown.md]: spelling...

File: docs/content/utilities/documentation/moose_flavored_markdown.md [hunk 2]
```diff
@@ -214,6 +216,31 @@ Markdown allowing an input code block to float to the right:
...
+
+An empty new line, designates the end of the css block.
+
+!css font-size=smaller margin-left=70% color=red text-shadow=1px 1px 1px rgba(0,0,0,.4)
+Another paragraph modified by CSS.
+```
+
+!css font-size=smaller margin-left=70% color=red text-shadow=1px 1px 1px rgba(0,0,0,.4)
+This paragraph should be of a smaller red font with a black offset text shadow. The text
+wil be aligned to the right due to the margin-left attribute (nice trick to preserve the
```
Comment (milljm) [inline docs/content/utilities/documentation/moose_flavored_markdown.md]: spelling...

**Comment by milljm** [comment]
Going to add !buildstatus to our markdown how to page and re-submit

File: python/MooseDocs/extensions/MooseBuildStatus.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+class MooseBuildStatus(MooseCommonExtension, Pattern):
+    """
+    Markdown extension for handling images.
+
+    Usage:
+      !image image_file.png|jpg|etc attribute=setting
+
+    All images/media should be stored in docs/media
+    """
```
Comment (aeslaughter) [inline python/MooseDocs/extensions/MooseBuildStatus.py]: This needs to be updated.

File: docs/content/utilities/documentation/moose_flavored_markdown.md [hunk 4]
```diff
@@ -214,6 +216,31 @@ Markdown allowing an input code block to float to the right:
...
+
+```markdown
+!css font-size=smaller margin-left=70% color=red text-shadow=1px 1px 1px rgba(0,0,0,.4)
+This paragraph should be of a smaller red font with a black offset text shadow. The text
+will be aligned to the right due to the margin-left attribute (nice trick to preserve the
+'justify' attribute currently in use)
+
+An empty new line, designates the end of the css block.
+
+!css font-size=smaller margin-left=70% color=red text-shadow=1px 1px 1px rgba(0,0,0,.4)
```
Comment (aeslaughter) [inline docs/content/utilities/documentation/moose_flavored_markdown.md]: This is awesome.

**Pull request merged**